POC_API/src/test/java/org/Execute.java

```diff
@@ -0,0 +1,23 @@
+package org;
+
+import org.utilities.FileManager;
+
+import java.util.List;
+import java.util.Map;
+
+public class Execute {
+    public static void main(String[] args) {
+        FileManager.createFileCopy("C:\\Users\\aturuk\\IdeaProjects\\mtf-automation-testing-api\\confirmClone.txt",
+                "C:\\Users\\aturuk\\IdeaProjects\\mtf-automation-testing-api\\confirmCloneDup.txt");
+
```

Comment:
You should be able to use relative paths so that other's can run this (they have different usernames).

Reply:
this is just sample class to test/debug  the  functions

Reply:
we don't really need it..